src/languages/en.ts

```diff
@@ -1336,6 +1336,7 @@ export default {
             notAuthorized: `You do not have access to this page. Are you trying to join the workspace? Please reach out to the owner of this workspace so they can add you as a member! Something else? Reach out to ${CONST.EMAIL.CONCIERGE}`,
             goToRoom: ({roomName}: GoToRoomParams) => `Go to ${roomName} room`,
             workspaceAvatar: 'Workspace avatar',
+            mustBeOnlineToViewMembers: 'You must be online in order to view members of this workspace',
```

Comment:
```suggestion
            mustBeOnlineToViewMembers: 'You must be online in order to view members of this workspace.',
```

https://github.com/Expensify/App/issues/27776#issuecomment-1751071519